setup

```diff
@@ -2,8 +2,15 @@
 
 import os
 import pip
+import argparse
 from shutil import copyfile, which
 
+parser = argparse.ArgumentParser(description='Setup serenata-de-amor using serenata-toolbox.')
```

Comment:
minor 💅 here: We are trying to keep our lines length < 80 as [PEP8 states](https://www.python.org/dev/peps/pep-0008/#maximum-line-length):

> Limit all lines to a maximum of 79 characters.